app/components/Admin/ReviewComment.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {Button, OverlayTrigger, Tooltip} from 'react-bootstrap';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faTrash} from '@fortawesome/free-solid-svg-icons';
+import {dateTimeFormat} from 'functions/formatDates';
+
+interface Props {
+  id: string;
+  description: string;
+  createdAt: string;
+  createdBy: string;
+  viewOnly: boolean;
+  isResolved: boolean;
+  onResolveToggle: (id: string, resolve: boolean) => void;
+  onDelete: (id: string) => void;
+}
+
+export const ReviewComment: React.FunctionComponent<Props> = ({
+  id,
+  description,
+  createdAt,
+  createdBy,
+  viewOnly,
+  isResolved,
+  onResolveToggle,
+  onDelete
+}) => {
+  return (
+    <li>
+      <div className="text-block">
+        <p>{description}</p>
+        <p className="timestamp">
+          <span>{createdBy}</span>
+          {`${dateTimeFormat(createdAt, 'minutes')}`}
+        </p>
+      </div>
+      {!viewOnly && (
+        <div>
+          <Button
+            size="sm"
+            variant="outline-primary"
+            style={{marginRight: '0.5em'}}
+            onClick={() => onResolveToggle(id, !isResolved)}
+          >
+            {`${isResolved ? 'Unresolve' : 'Resolve'}`}
+          </Button>
+          <OverlayTrigger
+            placement="top"
+            overlay={(props) => (
+              <Tooltip id="delete" {...props}>
+                Delete comment
+              </Tooltip>
+            )}
+          >
+            <Button
+              size="sm"
+              aria-label="Delete"
+              variant="secondary"
+              onClick={() => onDelete(id)}
```

Comment:
Do we want to show a confirmation modal here? `DeleteConfirmationModal` could be reused (with a bit of tweaking)

Reply:
The display of comment list items is borrowed from the previous design that didn't use that pattern, and comment deletion didn't really come up as an important point in our design process + feedback from our SME. However - requested by her, there **will** be one shown if you try to mark a review completed when there are outstanding unresolved comments.

All of the review step-related work was supposed to come in as part of [GGIRCS-2293](https://youtrack.button.is/issue/GGIRCS-2293) next up, so if necessary that may be an opportunity to do the same for comment deletion.